### PR TITLE
Added nutrition facts to SIW docs

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Added nutrition facts to SIW docs
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-407996](https://oktainc.atlassian.net/browse/OKTA-407996)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md [hunk 1]
```diff
@@ -5,9 +5,26 @@ title: Deprecated JavaScript methods in the Sign-In Widget
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-## Overview
+This guide covers the JavaScript methods that are deprecated from the Sign-In Widget and the sign-in methods that are used after you upgrade your org to Okta Identity Engine.
 
-After you upgrade your org to Okta Identity Engine, the [setCookieAndRedirect()](https://github.com/okta/okta-signin-widget#renderel) JavaScript method is deprecated from the Sign-In Widget. In the Classic Engine, your app integration calls the `setCookieAndRedirect()` method which means that your app integration sets the redirect URI. However, in the Identity Engine, your app integration shouldn’t assume whether it sets the redirect URI as part of the flow. Administrators set the sign-on policies in the [Okta Admin Console](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-policies). Instead of the `setCookieAndRedirect()` method, use the `showSignIn()` method to resolve the returned promise or redirect it, based on the Administrator’s policy.
+---
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:10]: Nitpick — include a blank line between the first horizontal rule and the "Learning outcomes" line.

Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:10]: Same comment for the other articles too.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md [hunk 2]
```diff
@@ -5,9 +5,26 @@ title: Deprecated JavaScript methods in the Sign-In Widget
...
+**Learning outcomes**
+
+Understand the `showSignIn` methods that are used in the Sign-In Widget.
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:13]: Might be nice to phrase this a bit more in terms of what the actual outcome is. Why does the reader need to understand these methods?

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-i18n/main/index.md [hunk 3]
```diff
@@ -5,15 +5,32 @@ title: Updates to the Sign-In Widget I18n properties
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-## Overview
+This guide covers how to specify i18n translation strings after your org upgrades to the Okta Identity Engine.
+
+---
+**Learning outcomes**
+
+Understand how to override text strings with Identity Engine i18n strings.
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-i18n/main/index.md:13]: Again, phrase in terms of actual outcome? For example, add something to the end of the sentence like "...to create localized sign-in widgets." or something?

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 4]
```diff
@@ -5,42 +5,61 @@ title: Upgrade your Okta Sign-In Widget
...
 <ApiLifecycle access="Limited GA" />
 
-## Overview
+
+This guide covers how to upgrade the Okta Sign-In Widget, which depends on whether your user authentication deployment model is a redirect authentication or an embedded authentication.
+
+---
+**Learning outcomes**
+
+* Understand the benefits of keeping your Sign-In Widget up to date.
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:14]: should "up to date" be written "up-to-date"? Usually is, in my experience, but I wonder if this is a style guide thing.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 5]
```diff
@@ -5,42 +5,61 @@ title: Upgrade your Okta Sign-In Widget
...
+* Update your Sign-In Widget based on your user deployment model.
+* Know how to obtain the latest version of the Sign-In Widget.
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:16]: It seems to me that logically, this should be the second bullet in the list — you understand why, then you learn how to obtain it, then you go and obtain it and update it.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md [hunk 6]
```diff
@@ -29,11 +46,11 @@ The following code sample shows how to use `showSignIn()` to handle a token flow
...
 ```javascript
 var signIn = new OktaSignIn({
-   // Assumes there is an empty element on the page with an id of ‘osw-container’
-  el: ‘#osw-container’,
+   // Assumes there is an empty element on the page with an id of 'osw-container'
+  el: '#osw-container',
   clientId: '${clientId of your OIDC app integration}',
   redirectUri: '${redirectUri configured in your OIDC app integration}',
-  baseUrl: ‘https://${yourOktaDomain},
+  baseUrl: 'https://${yourOktaDomain},
```
Comment (chrisdmills-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:53]: this property value is missing a closing quote mark

**Review by chrisdmills-okta — CHANGES_REQUESTED** [review]
Hi @christinechoy-okta ! Looking pretty good. I found a few nitpicks, but nothing too major.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md (hunk 4 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:14]: @chrisdmills-okta it is hyphenated if it precedes a noun. It's not hyphenated if it comes after a noun.
Example:
You can keep an up-to-date Sign-In Widget.
You can keep your Sign-In Widget up to date.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md (hunk 6 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:53]: Fixed

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:10]: Fixed.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md (hunk 2 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:13]: Fixed

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-i18n/main/index.md (hunk 3 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-i18n/main/index.md:13]: Fixed

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md (hunk 5 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:16]: Fixed

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md (hunk 4 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:14]: Ah, that makes sense — thanks for clarifying!

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md (hunk 6 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget-deprecated-methods/main/index.md:37]: This still doesn't look fixed. Also, below here there now looks to be a merge conflict in the file (line 38), and backticks inserted in the file (line 40). Is your text editor doing something weird in that regard?

**Review by chrisdmills-okta — APPROVED** [review]
Looks mostly great now @christinechoy-okta . I did have a couple more concerns for you to check up on in one of the files, but I'm also approving it so that it doesn't block you from merging it after fixing.

**Pull request merged**